src/main/java/frc/robot/subsystems/CDSSubsystem.java

```diff
@@ -54,6 +60,42 @@ public void stopCDS() {
     CDSWheelControllerOne.getSparkMax().set(0.0);
     CDSBeltController.getSparkMax().set(0.0);
     SmartDashboard.putNumber("CDS Belt Speed", 0.0);
+  }
+  
+  public int[] getSensorStatus() {
+    int frontStatus = colorSensorOne.getProximity() > 800 ? 1: 0;
+    int backStatus = backBeamBreak.get() ? 1: 0;
+    int[] beamBreakArray = {frontStatus, backStatus};
+    return beamBreakArray;
+  }
+  
+  public void periodic() {
+    // Color sensing
+    String ballColor = senseColor();
+    String allianceColor = getAllianceColor();
```

Comment:
May eventually just want to publish this once to avoid adding noise, since it will not change once the Robot code starts